test/production/app-dir/mangle-reserved/mangle-reserved.test.ts

```diff
@@ -5,8 +5,8 @@ createNextDescribe(
   {
     files: __dirname,
   },
-  ({ next }) => {
-    it('should work using cheerio', async () => {
+  ({ next, isTurbopack }) => {
+    ;(isTurbopack ? it.skip : it)('should work using cheerio', async () => {
```

Comment:
What's the random semicolon doing here?

Reply:
I'm not sure. I saw lots of tests had this so I just followed. I think it's about preventing ASI.